assets/questions/findChar.js

```diff
@@ -0,0 +1,114 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Reverse Words.
+ */
+
+globalData.questions['findChar'] = { // eslint-disable-line dot-notation
+  title: 'Find the first character in a string that doesn\'t repeat',
+  starterCode: {
+    python: `def find1stNonRepeatedChar(word):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python: `class AuxiliaryCode(object):
+    @classmethod
+    def sortChar(cls, word):
+        sortedStr = ''.join(sorted(word))        
+        c = [a for a in sortedStr if sortedStr.count(a) == 1]
+        if not c:
+            return None
+        else:
+            return c[0]
+`
+  },
+  tasks: [
+    {
+      instructions: [
+        [
+          'For this question, you\'ll implement the find1stNonRepeatedChar function. ',
+          'Write a function to find the first character in a String that doesn\'t repeat.'
+        ].join(''),
+        'For instance, \'mom\' would output \'o\'.'
+      ],
+      prerequisiteSkills: ['Strings', 'String Manipulation'],
+      acquiredSkills: ['String Manipulation'],
+      inputFunctionName: null,
+      outputFunctionName: null,
+      mainFunctionName: 'find1stNonRepeatedChar',
+      correctnessTests: [
+        {input: 'mom', 
```

Comment:
This is not consistent with the style of other questions. Please update these dicts to be consistent with the other questions that are already in the codebase.